src/main/java/uk/org/tombolo/importer/ons/CensusImporter.java

```diff
@@ -43,12 +44,32 @@
 
     @Override
     public List<String> getDatasourceIds() {
-        return getDataSourceIDs();
+        try {
+            return getSeedData().stream().map(CensusDescription::getDataSetTable)
+                    .collect(Collectors.toCollection(ArrayList::new));
+        } catch (IOException e) {
+            log.error("An error has occurred while downloading DatasourceID's" + e.getMessage());
+        }
+
+        return Collections.emptyList();
+    }
+
+    @Override
+    protected List<String> getOaDatasourceIds() {
+        return subjectRecipes.stream().filter(subjectRecipe ->
+            EnumUtils.isValidEnum(OaImporter.OaType.class, subjectRecipe.getSubjectType())).map(subjectRecipe ->
+                subjectRecipe.getSubjectType()).collect(Collectors.toCollection(ArrayList::new));
     }
```

Comment:
I believe with this change we could now remove 

```
        "importerClass": "uk.org.tombolo.importer.ons.OaImporter",
        "datasourceId": "lsoa"
```
from renting-percentiles-for-leeds-normalised-nationally.json.
However, upon doing that and running the recipe the system didn't try to download the lsoa boundaries. But when added the above code segment back to datasources, DC did download the lsoa boundaries.

Please let me know if this is not how this was suppose to be used.

Reply:
Checking this now.

Reply:
aha, got it :dancing_men: . making the changes.